src/edge/views.py

```diff
@@ -226,6 +226,29 @@ def on_post(self, request, fragment_id):
         return {}, 201
 
 
+class FragmentAnnotateChunksView(ViewBase):
+
+    @transaction.atomic()
+    def on_post(self, request, fragment_id):
+        annotation_parser = RequestParser()
```

Comment:
is there a good reason to keep this pattern over using forms?

Reply:
no, just wanted to stay the same as before